.github/workflows/run-tests.yml

```
@@ -0,0 +1,238 @@
+        env:
+          fail-fast: true
+
+      - name: Copy Coveralls
+        id: copy_coveralls
+        if: success()
+        run: docker cp "$(docker-compose -f .github/tests/docker-compose.yml ps -q drupal)":/srv/www/html/build/logs/clover.xml . || echo "skip=true" >> "$GITHUB_OUTPUT"
+
+      - name: Monitor coverage
+        id: coveralls
+        if: ${{ steps.copy_coveralls.outputs.skip != 'true' }}
+        uses: slavcodev/coverage-monitor-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          coverage_path: "clover.xml"
+          threshold_alert: 0
+          threshold_warning: 50
+          threshold_metric: "lines"
+          comment_footer: false
+
+      - name: DB Logs
+        id: dblog
+        if: failure()
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker-compose -f .github/tests/docker-compose.yml exec -T drupal drush watchdog:show
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Build output
+
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ### Build output
+            #### Composer Validate `${{ steps.validate.outcome }}`
+            #### PHP Lint `${{ steps.lint.outcome }}`
+            #### Docker Build `${{ steps.build.outcome }}`
+            #### Environment Setup `${{ steps.docker.outcome }}`
+            #### Site Install `${{ steps.install.outcome }}`
+            #### PHP Code Sniffer `${{ steps.phpcs.outcome }}`
+
+            <details><summary>Software Versions</summary>
+            <code>${{ steps.versions.outputs.stdout }}</code>
+            </details>
+            <details><summary>Drupal Logs</summary>
+            <code>${{ steps.dblog.outputs.stdout }}</code>
+            </details>
+
+            *Pusher: @${{ github.actor }}, Action: `${{ github.event_name }}`, Workflow: `${{ github.workflow }}`*
+          edit-mode: replace
+
+      - name: Slack Success Notification
+        id: slack_success
+        if: success()
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: '${{ secrets.SLACK_CHANNEL }}'
```

I'm slowly replacing this, so I have a non-secret value I can actually see in the UI :-)

```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```